frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -109,6 +128,11 @@
     </v-skeleton-loader>
     <CancelApplicationDialog :show="showCancelDialog" :applicationId="cancelledApplicationId" :applicationType="applicationTypeToCancel" @close="toggleCancelDialog" @cancel="cancelApplication" />
     <AppBackButton id="back-home-button" width="220px" :to="{ name: 'home' }">Home</AppBackButton>
+    <NewRequestDialog
+      class="pa-0"
```

Comment:
I see this class in a few places but it doesn't seem to impact the layout at all so it can be removed.